rules/integrations/google_workspace/defense_evasion_google_workspace_new_oauth_login_from_third_party_application.toml

```diff
@@ -47,7 +47,7 @@ type = "new_terms"
 
 query = '''
 event.dataset: "google_workspace.token" and event.action: "authorize" and
-google_workspace.token.scope.data.scope_name: *Login and google_workspace.token.client.id: *apps.googleusercontent.com
+google_workspace.token.scope.data: *Login and google_workspace.token.client.id: *apps.googleusercontent.com
```

Comment:
@gpapakyriakopoulos can you update the `updated_date` value for this rule.
Likely unit tests are failing because `google_workspace.token.scope.data` needs to be added to the `non-ecs.json` file under `detection-rules/etc/non-ecs.json`.

Can you add that in and map it to the flattened data type since you have a fork of the repository.

Reference: https://github.com/elastic/detection-rules/blob/ec6038b9d9176ea2d14481fba5bc777b9e954fb4/detection_rules/etc/non-ecs-schema.json#L138

Reply:
Done!